docs/system_overview/configuration.md

```diff
@@ -347,6 +347,11 @@ This file defines the behavior of the server Agent that answers individual reque
         ]
     },
 
+    // (Optional)
+    // The absolute path to a directory which the server will load plugins from.
+    // Overrides the default directory.
```

Comment:
Is it worth documenting how one can find out what the default directory is?

Reply:
I don't think so. We use the standard OS locations and figuring out that info seems like a basic skill all admins must have.

Reply:
Okay, just making sure. Resolving.